Cargo.toml

```diff
@@ -32,7 +32,9 @@ nix = { version = "0.26", default-features = false, features = [
     "user",
 ] }
 # Used for parsing procfs info.
-procfs-core = { git = "https://github.com/eminence/procfs", rev = "591016286d195f523628eccf5cac9c923ae6ea45" }
+procfs = "0.15.1"
+# TODO: move to when it is published as it reduces dependencies
+#procfs-core = { git = "https://github.com/eminence/procfs", rev = "591016286d195f523628eccf5cac9c923ae6ea45" }
```

Comment:
I'd rather have us revert https://github.com/rust-minidump/minidump-writer/commit/b95bf2e31f0be429a78f9639835312d8423fd632 instead of falling back on vanilla `procfs` because we can't vendor it in Firefox due to its broad dependencies (including windows-sys which we can't still vendor). If that's not a problem I'll re-land the switch to `procfs-core` as soon as they cut a proper release.

Reply:
Ahh ok I'll wait for a response from the maintainers (probably US? timezone) today and if they can't make a release can revert that.